package.json

```diff
@@ -28,7 +28,7 @@
     "@nestjs/passport": "7.1.0",
     "@nestjs/platform-express": "7.4.4",
     "@nestjs/swagger": "4.6.1",
-    "@nestjs/typeorm": "7.1.4",
+    "@nestjs/typeorm": "^7.1.4",
```

Comment:
I think you can leave out the `^` as before

Reply:
I dont know why there are 2 of the same modules. I didnt add any for this.